config/config.exs

```diff
@@ -35,6 +35,9 @@ config :hexpm,
   fastly_key: System.get_env("HEX_FASTLY_KEY"),
   fastly_hexdocs: System.get_env("HEX_FASTLY_HEXDOCS"),
   fastly_hexrepo: System.get_env("HEX_FASTLY_HEXREPO"),
+  billing_impl: Hexpm.Billing.Hexpm,
```

Comment:
consider choosing implementation similarly how we do it for store & cdn at the top of the file

```elixir
billing = if System.get_env("HEX_BILLING_URL"), do: Hexpm.Billing.Hexpm, else: Hexpm.Billing.Local
```